src/aiai_eval/utils.py

```diff
@@ -114,8 +116,6 @@ def log_scores(
     for metric_cfg in metric_configs:
         agg_scores = aggregate_scores(scores=scores, metric_config=metric_cfg)
         test_score, test_se = agg_scores["test"]
-        test_score *= 100
-        test_se *= 100
 
         msg = f"{metric_cfg.pretty_name}:\n  - Test: {test_score:.2f} ± {test_se:.2f}"
```

Comment:
If we don't multiply the scores by 100 then this should be changed to .4f, as two decimals won't give enough detail; it'll only state things like 0.54 instead of 0.5423.

Reply:
Agreed.